DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/BasicAuthenticatedAccessibilityTests.cs

```diff
@@ -60,7 +60,7 @@ public class BasicAuthenticatedAccessibilityTests : AccessibilityTestsBase,
         [InlineData("/NotificationPreferences/Edit/AdminUser", "Update notification preferences")]
         [InlineData("/NotificationPreferences/Edit/DelegateUser", "Update notification preferences")]
         [InlineData("/ChangePassword", "Change password")]
-        [InlineData("/TrackingSystem/Support", "Support")]
+        [InlineData("/TrackingSystem/Delegates/Groups/103/EditDescription", "Edit description for ABC Digital Supervision 2 group (optional)")]

```

Comment:
The accessibility test is failing because group 103 is not in the base version of the database, so we have different ones. Use group 5 instead since we're using that for all the other tests and move this next to the other similar tests (lines 46-51).

Also you've removed the Support test, this needs bringing back